.github/community-cc-list.md

```diff
@@ -0,0 +1,14 @@
+<!--
+PHP_CodeSniffer Community CC List
+
+This list will be used on select tickets to ping the wider PHP_CodeSniffer community for input.
+
+If you want to be on this list, feel free to submit a PR to add yourself.
+If you want to be removed from the list, dito, submit a PR to remove yourself and it will be merged without questions.
+
+PRs adding/removing other people to/from the list will only be merged if the people being added/removed leave a comment on the PR consenting to it.
+
+For more context, see https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/227
+-->
+
+/cc @kukulich, @greg0ire @Ocramius
```

Comment:
The comma/space was a typo from my side which I've corrected.

As for putting each handle on a separate line - good point.

I'm a bit in doubt though on whether to do that as yes, it will make a PR diff easier to review, but on the other hand, it will make _using_ the list more cumbersome as I'd normally want to copy/paste this as a one-liner in the ticket text and having each user on a separate line would require editing the paste to remove the new lines.

Reply:
I think just using space with the list would look a bit more like the [codeowners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) file, so I'd probably opt to follow that convention as well.